clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -49,9 +49,9 @@
      *
      *  UNKNOWN_TOPIC_OR_PARTITION (3)  <- only for request v0
      *  OFFSET_LOAD_IN_PROGRESS (14)
-     *  NOT_COORDINATOR_FOR_CONSUMER (16)
```

Comment:
This is not introduced in this patch, but why we are returning GROUP_COORDINATOR_NOT_AVAILABLE for OffsetCommit if (!isActive.get), while return NOT_COORDINATOR_FOR_GROUP for OffsetFetch for the same condition? Do you know if there is any motivation?